lib/fake_artist/table.ex

```diff
@@ -243,4 +283,37 @@ defmodule FakeArtist.Table do
       current_seat + 1
     end
   end
+
+  @spec everyone_has_voted(map()) :: boolean()
+  defp everyone_has_voted(state) do
+    game_master_vote = 1
+
+    Enum.count(state.players |> Map.values(), fn x -> x.voted_for == nil end) ==
```

Comment:
Btw, `Map`'s are already Enumerable, so you can `Enum.count` directly like
```elixir
Enum.count(state.players, fn {_k, v} -> v.voted_for == nil end)
```
what you have is fine, but might be slightly less efficient since you're turning it into a List explicitly first (though I honestly have no idea what the reality is)

Reply:
I like your way a little better